doc/manual/manual.tex

```diff
@@ -945,19 +989,18 @@ \subsubsection{Coefficient self-consistency}
 \begin{equation}
   C_p(p, T)
   = C_p(p_{\textrm{ref}}, T) -T \int_{p_{\textrm{ref}}}^p
-  \pddc{\left(\alpha / \rho \right)}{T}{p}
+  \left(\frac{\partial \left(\alpha / \rho \right)}{\partial T} \right)_{p}
   \text{d}p.
 \end{equation}
-There is no guarantee that this expression will have a simple form for
-which the integral can be found analytically. 
+There is no guarantee that any particular $\rho(p, T)$ equation of state will yield a simple form for $C_p(p, T)$.
```

Comment:
Can you include the part `for which the integral can be found analytically` again? I think @bangerth added that after you had written the text. I assume you just removed it accidentally, or was there anything wrong with it?

Reply:
Yes, I can re-add. Nothing wrong with it, just too many thoughts in one sentence, and I forgot that @bangerth had added something here. I honestly thought I was going mad. 

("Only *going* mad?", I hear you say? "That ship sailed long ago...")